react-app/src/native/components/register/Comment.js

```diff
@@ -0,0 +1,212 @@
+                                <Picker.Item label={"Choose Method"} value={''} key={0} />
+                                <Picker.Item label={"I will send an money email transfer to dave@perpetualmotion.org"} value={'I will send an money email transfer to dave@perpetualmotion.org'} key={1} />
+                                <Picker.Item label={"I will mail a cheque to the Perpetual Motion head office"} value={'I will mail a cheque to the Perpetual Motion head office'} key={2} />
+                                <Picker.Item label={"I will bring a cash/cheque to the Perpetual Motion head office"} value={'I will bring a cash/cheque to the Perpetual Motion head office'} key={3} />
+                                <Picker.Item label={"I will bring cash/cheque to registration night"} value={'I will bring cash/cheque to registration night'} key={4} />
+                                
+                                
+                            </Picker>
+                        </View>
+                    </View>
+                    
+                    <Text style={ [styles.normalText, {fontWeight:'bold'}]}>Make Checks Payable to Perpetual Motion</Text>
+                    <Text style={ [styles.normalText, {fontWeight:'bold'}]}>Send This Confirmation Form & Fees to:</Text>
+                    <Text style={styles.normalText}>78 Kathleen St. Guelph, Ontario; H1H 4Y3</Text>
+                </View>
+
+                <Text style={styles.header}>Registration Due By</Text>
+                <View style= {styles.line}/>
+                <View style={styles.addPadding}>
+                    <Text style={ [styles.normalText, {fontWeight:'bold'}]}>Spring League</Text>
+
+                    <Text style={styles.normalText}>Ultimate Frisbee
+                        <Text style={{color:'#FF0000'}}>    *Insert date Here*</Text>
+                    </Text>
+
+                    <Text style={styles.normalText}>Beach Volleyball
+                        <Text style={{color:'#FF0000'}}>    *Insert date Here*</Text>
+                    </Text>
+
+                    <Text style={styles.normalText}>Flag Football
+                        <Text style={{color:'#FF0000'}}>    *Insert date Here*</Text>
+                    </Text>
+
+                    <Text style={styles.normalText}>Soccer
+                        <Text style={{color:'#FF0000'}}>    *Insert date Here*</Text>
+                    </Text>
+
+                </View>
+
+                <Text style={styles.header}>Register</Text>
+                <Text style = {styles.subHeading}>Submit your group registration to the convenor.</Text>
+                <View style= {styles.line}/>
+                <View style={styles.addPadding, {justifyContent:'space-between', flexDirection:'row'}}>
+                    <Button title={'Register'} color='red' style={{border:1}} onPre/>
+                    <Button title={'Print form?'} color='red' onPress={() => {
+                        console.log("JERE")
```

Comment:
i think it's safe enough to not have a print button here. If they need to print a form they can do it from the website.